modules/api-support-tiers.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * rest_api/understanding-api-support-tiers.adoc
+
+[id="api-tiers_{context}"]
+= API tiers
+
+All commercially supported APIs, components, and features are associated under one of the following support levels:
+
+[discrete]
+[id="api-tier-1_{context}"]
+== API tier 1
+APIs and application operating environments (AOEs) are stable within a major release for a minimum of 12 months or 3 minor releases from the announcement of deprecation, whichever is longer. The release that introduces a new or revised API or AOE, and the two following minor releases (n, n+1, n+2).
+
+[discrete]
+[id="api-tier-2_{context}"]
+== API tier 2
+APIs and AOEs are stable within a major release for a minimum of 9 months or 3 minor releases from the announcement of deprecation, whichever is longer. API tier 2 is the default for components in {product-title} where not identified.
```

Comment:
this claim that tier 2 is the default when not specified contradicts the claim further down that:
```
Any API group that does not specify an explicit compatibility level is assigned API tier 3 by default.
```

Reply:
lets remove that sentence, it confuses me now.